### ISSUE TITLE
OSX-["NEW_FUN_TAGS"] issue

### ISSUE THREAD

**Issue by bmeric** [massemanet/distel#63]
Hi,

When i try to use erl-complete i get this error on Erlang side;

`Connection attempt from node 'distel_18982@Barss-MacBook-Pro.local' rejected since it cannot handle ["NEW_FUN_TAGS"].**`

And `"nodedown: emacs@Barss-MacBook-Pro\.local"` message on emacs side.

I also get `"Buffer "*derl emacs@Barss-MacBook-Pro.local*" has a running process; kill it? (yes or no)"` message. 

I'm using Emacs 26.1, Erlang/OTP 21 and latest git version of Distel. My .emacs is;

```elisp
(add-to-list 'load-path "~/emacsbm/distel/elisp")
(require 'distel)
(distel-setup)

;; prevent annoying hang-on-compile
(defvar inferior-erlang-prompt-timeout t)
;; default node name to emacs@localhost
(setq inferior-erlang-machine-options '("-sname" "emacs"))
;; tell distel to default to that node
(setq erl-nodename-cache
      (make-symbol
       (concat
        "emacs@"
        ;; Mac OS X uses "name.local" instead of "name", this should work
        ;; pretty much anywhere without having to muck with NetInfo
        ;; ... but I only tested it on Mac OS X.
                (car (split-string (shell-command-to-string "hostname"))))))
```

Is there anything wrong with my configuration or my setup?

Thanks in advance.

**Comment by massemanet** [comment]
must be a new wrinkle in erlang distribution introduced in 21.

**Comment by bmeric** [comment]
Oh, you're right. erl_complete works as expected when i downgrade Erlang to 20.

Thanks.

**Comment by massemanet** [comment]
fixed in 6fe41f8
